.github/workflows/update.yaml

```diff
@@ -0,0 +1,25 @@
+name: Docker Base Image CI
+
+concurrency:
+  group: docker-base-image-ci
+  cancel-in-progress: true
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v3
+      - name: Check for image updates
+        id: check
+        uses: lucacome/docker-image-update-checker@v1
+        with:
+          base-image: library/ubuntu:22.04
+          image: josecols/meta:3.0.2
+      - name: Abort
```

Comment:
By aborting this workflow, we signal to the main action, `Docker Image CI`, that there are no pending updates and avoid execution.